packages/swarm-components/package.json

```diff
@@ -1,53 +1,60 @@
 {
-  "name": "@meetup/swarm-components",
-  "version": "0.6.5",
-  "description": "Web Components for Swarm",
-  "author": "hsbacot <hsbacot@gmail.com>",
-  "homepage": "https://github.com/meetup/swarm-ui#readme",
-  "license": "ISC",
-  "main": "lib/index.js",
-  "directories": {
-    "lib": "lib",
-    "test": "__tests__"
-  },
-  "files": [
-    "lib"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/meetup/swarm-ui.git"
-  },
-  "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1",
-    "build": "babel src/ -d lib/",
-    "develop": "babel src/ --watch -d lib/"
-  },
-  "bugs": {
-    "url": "https://github.com/meetup/swarm-ui/issues"
-  },
-  "bundlesize": [
-    {
-      "path": "lib/*.js",
-      "maxSize": "> 10%"
-    }
-  ],
-  "peerDependencies": {
-    "react": "^16.6.3"
-  },
-  "devDependencies": {
-    "@babel/cli": "7.4.3",
-    "@babel/core": "7.4.3",
-    "@babel/plugin-proposal-class-properties": "7.4.0",
-    "@babel/preset-env": "7.4.3",
-    "@babel/preset-flow": "7.0.0",
-    "@babel/preset-react": "7.0.0",
-    "babel-plugin-react-docgen": "2.0.2",
-    "flow-bin": "0.96.0"
-  },
-  "dependencies": {
-    "@meetup/swarm-constants": "^0.2.9",
-    "classnames": "2.2.6",
-    "html-loader": "0.5.5",
-    "swarm-icons": "3.8.354"
-  }
+	"name": "@meetup/swarm-components",
```

Comment:
I know it's a little annoying, but `.json` files should use spaces instead of tabs, especially `package.json`, which won't work with tabs in npm before version 5 https://github.com/prettier/prettier/issues/3533

Not sure if this was a Prettier change, but we should make sure json files get 2-space indentation

Reply:
cc @hsbacot, looks like we keep changing tabs/spaces back and forth.

Do we have a standard for that? do we have a config we can just reuse?

Reply:
Looks like mup-web just solves this with a `.prettierignore` file https://github.com/meetup/mup-web/blob/master/.prettierignore

Reply:
Hmm. It looks like this repo has plenty of things in `.prettierignore`: https://github.com/meetup/swarm-ui/blob/master/.prettierignore

@hsbacot, is this all intentional here?